code/drasil-lang/lib/Language/Drasil/NounPhrase/Core.hs

```diff
@@ -2,22 +2,32 @@
 module Language.Drasil.NounPhrase.Core (
   -- * Types
   CapitalizationRule(..), NP(..),
-  PluralForm, PluralRule(..)) where
+  PluralForm, PluralRule(..),
+  NPStruct(S,(:-:),(:+:),P)) where
 
-import Language.Drasil.Sentence (Sentence)
+import Language.Drasil.Symbol (Symbol)
 
--- | Synonym for 'Sentence' typically used for plural forms.
-type PluralForm = Sentence  -- These might change.
+-- | Essentially a subset of 'Sentence' that contains only the parts
+-- that make sense for a NounPhrase
+data NPStruct =
+    S String
+  | NPStruct :-: NPStruct -- no space
+  | NPStruct :+: NPStruct -- a space
+  | P Symbol
```

Comment:
Neat idea! I was thinking we would jump back to `String`, but this is probably better.